src/test/java/org/inferred/freebuilder/processor/FeatureSets.java

```diff
@@ -0,0 +1,42 @@
+package org.inferred.freebuilder.processor;
+
+import static org.inferred.freebuilder.processor.util.feature.SourceLevel.JAVA_6;
+import static org.inferred.freebuilder.processor.util.feature.SourceLevel.JAVA_7;
+
+import com.google.common.collect.ImmutableList;
+
+import org.inferred.freebuilder.processor.util.feature.FeatureSet;
+import org.inferred.freebuilder.processor.util.feature.FunctionPackage;
+import org.inferred.freebuilder.processor.util.feature.GuavaLibrary;
+import org.inferred.freebuilder.processor.util.feature.StaticFeatureSet;
+
+import java.util.List;
+
+final class FeatureSets {
+
+  /** For tests valid in any environment. */
+  public static final List<FeatureSet> ALL = ImmutableList.of(
+      new StaticFeatureSet(JAVA_6),
+      new StaticFeatureSet(JAVA_7),
+      new StaticFeatureSet(JAVA_7, FunctionPackage.AVAILABLE),
```

Comment:
No, I just didn't want to create a god object so separated out FunctionPackage, and there's been no other Java 8 features I've been switching on yet so JAVA_7 actually means JAVA_8 too.